swan-cern/templates/fluentd/fluentd_sources.conf.yaml

```diff
@@ -0,0 +1,42 @@
+{{ if .Values.fluentd.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluentd-sources-conf
+  namespace: {{ .Release.namespace }}
+data:
+  sources.conf: |-
+      # Do not collect fluentd logs to avoid infinite loops.
+      <match kubernetes.var.log.containers.**fluentd**>
+          @type null
+      </match>
+
+    {{ if not .Values.fluentd.output.includeInternal }}
+      <match kubernetes.var.log.containers.**_kube-system_**>
+          @type null
+      </match>
+    {{ end }}  
+
+      # Read from container logs
+      <source>
```

Comment:
This source is to specify our main source of logs, and we need to do this here now because before it was done by the old fluentd chart?

Reply:
Yes, that is correct. The official upstream chart does not have this defined by default.